components/core/FollowButton/FollowButton.test.ts

```diff
@@ -32,7 +32,7 @@ describe('<FollowButton />', () => {
       it('should call the default onSubmit methd', async () => {
         const { container } = stubbedRender(FollowButton);
 
-        await fireEvent.click(container.firstChild);
+        fireEvent.click(container.firstChild as HTMLElement);
```

Comment:
Not a fan with using as when there is an alternative, as you are forcing a type here.

Instead you can use casting and do:
```suggestion
        fireEvent.click(<HTMLElement>container.firstChild);
```